Source/WebCore/html/PermissionsPolicy.cpp

```diff
@@ -116,6 +118,7 @@ static std::pair<PermissionsPolicy::Feature, StringView> readFeatureIdentifier(S
 #endif
 #if ENABLE(WEB_AUTHN)
     constexpr auto publickeyCredentialsGetRuleToken { "publickey-credentials-get"_s };
+    constexpr auto digitalCredentialsGetRuleToken { "digital-credentials-get"_s };
```

Comment:
Is this tightly coupled with WEBAUTHN?

Reply:
Yeah, because of the Credential Management... we need to decouple them all three (Cred Man, Web Auth, Digital Cred)... hi @pascoej 👋